salt/states/reg.py

```diff
@@ -192,9 +192,14 @@ def present(name,
                                    salt.utils.to_unicode(name, 'utf-8'))
         return ret
 
+    try
```

Comment:
syntax error, need a colon... _try`:`_

Reply:
Yes, thanks for the catch. I was working on several things at once and clearly rushed this PR

Reply:
No judgement! Was just testing the change on my box and of course this came up quickly. I make a thousand such mistakes in the course of fixing things. No explanation needed. 😁